tachyon/math/elliptic_curves/short_weierstrass/affine_point.h

```diff
@@ -233,6 +234,33 @@ class Copyable<math::AffinePoint<
   }
 };
 
+template <typename Curve>
+class RapidJsonValueConverter<math::AffinePoint<
+    Curve,
+    std::enable_if_t<Curve::kType == math::CurveType::kShortWeierstrass>>> {
+ public:
+  using Field = typename math::AffinePoint<Curve>::BaseField;
+
+  template <typename Allocator>
+  static rapidjson::Value From(const math::AffinePoint<Curve>& value,
+                               Allocator& allocator) {
+    rapidjson::Value object(rapidjson::kObjectType);
+    AddJsonElement(object, "x", value.x(), allocator);
+    AddJsonElement(object, "y", value.y(), allocator);
+    return object;
+  }
+
+  static bool To(const rapidjson::Value& json_value, std::string_view key,
```

Comment:
```suggestion
  [[nodiscard]] static bool To(const rapidjson::Value& json_value, std::string_view key,
```

Reply:
Is it necessary to include the `nodiscard` keyword in utility functions? In my opinion, I'd prefer not to include it for user's convenience.